web/content/docs/benchmarks/thermo-mechanics/thermomechanics.md

```diff
@@ -0,0 +1,38 @@
++++
+project = "https://github.com/ufz/ogs-data/blob/master/ThermoMechanics/cube_1e3.prj"
+author = "Xing-Yuan Miao"
+date = "2017-05-08T15:10:33+01:00"
+title = "Thermoelastic Stress"
+weight = 156
+
+[menu]
+  [menu.benchmarks]
+    parent = "thermo-mechanics"
+
++++
+
+{{< project-link >}}
+
+## Problem description
+
+We solve a thermo-mechanical homogeneous model in cube domain. The dimensions of this cube model are 1\,m in all directions. The boundary conditions and temperature loadings, as well as the material can refer Chapter 14 in Kolditz et al. for detailed problem description.
+
+## Results and evaluation
+
+Result showing temperature and stresses development with time in the centre node of the model:
+
+{{< img src="../temperature.png" >}}
+{{< img src="../stress.png" >}}
+
```

Comment:
use a log-scale (on x) for the images or something similar ... right now, the scaling makes a comparison difficult.